model/map/src/main/java/org/keycloak/models/map/user/MapUserProvider.java

```diff
@@ -336,7 +333,7 @@ public UserModel addUser(RealmModel realm, String id, String username, boolean a
         entity.setUsername(username.toLowerCase());
         entity.setCreatedTimestamp(Time.currentTimeMillis());
 
-        entity = tx.create(entity);
+        tx.create(entity);
```

Comment:
It seems that on this line we had `entity =` after the first commit and then the second commit removed it. Should `entity =` remain there?